airQualityMonitoring/jsonld-contexts/airQualityMonitoring.jsonld

```diff
@@ -0,0 +1,6 @@
+{
+  "@context": [
+    "https://raw.githubusercontent.com/smart-data-models/dataModel.Environment/master/context.jsonld",
```

Comment:
I don't see your `AirQuality` type in it?

Reply:
or you plan to rename it to `AirQualityObserved` (which is OK), which is unfortunately named `airQualityObserved` in https://raw.githubusercontent.com/smart-data-models/dataModel.Environment/master/context.jsonld

Reply:
ok but how can i know what is the expanded type of AirQuality ?
I suppose I need to create a new context like this, but don't know the '???'
{
   "@context": {
      "AirQuality": "https://???",
   }
}

Reply:
actually, `AirQualityObserved` is better than `AirQuality`. And use the same expanded form as in https://raw.githubusercontent.com/smart-data-models/dataModel.Environment/master/context.jsonld since you are based on it.